doc/source/f2py/distutils.rst

```diff
@@ -26,30 +27,32 @@ sources, call F2PY to construct extension modules, etc.
 
   __ setup_example.py
 
+Extensions to ``distutils``
+===========================
+
 :mod:`numpy.distutils` extends ``distutils`` with the following features:
 
 * :class:`Extension` class argument ``sources`` may contain Fortran source
   files. In addition, the list ``sources`` may contain at most one
-  F2PY signature file, and then the name of an Extension module must
-  match with the ``<modulename>`` used in signature file.  It is
+  F2PY signature file; in this case, the name of an Extension module must
+  match with the ``<modulename>`` used in signature file. It is
   assumed that an F2PY signature file contains exactly one ``python
   module`` block.
 
-  If ``sources`` does not contain a signature files, then F2PY is used
-  to scan Fortran source files for routine signatures to construct the
-  wrappers to Fortran codes.
+  If ``sources`` do not contain a signature file, then F2PY is used to scan
+  Fortran source files to construct wrappers to the Fortran codes.
 
-  Additional options to F2PY process can be given using :class:`Extension`
-  class argument ``f2py_options``.
+  Additional options to the F2PY executable can be given using the
+  :class:`Extension` class argument ``f2py_options``.
 
 * The following new ``distutils`` commands are defined:
 
   ``build_src``
     to construct Fortran wrapper extension modules, among many other things.
   ``config_fc``
-    to change Fortran compiler options
+    to change Fortran compiler options.
 
-  as well as ``build_ext`` and  ``build_clib`` commands are enhanced
+  as well as ``build_ext`` and  ``build_clib`` commands which are also enhanced
```

Comment:
Maybe "as well as the ``build_ext`` and  ``build_clib`` commands that are also ..."

Reply:
Does
>  Additionally, the ``build_ext`` and ``build_clib`` commands are also enhanced
  to support Fortran sources.
Sound ok?